megamek/src/megamek/common/UnitType.java

```diff
@@ -98,5 +98,26 @@ public static String getTypeDisplayableName(int type) {
         }
         throw new IllegalArgumentException("Unknown unit type");
     }
-
+    
+    // series of convenience methods to shorten unit type determination
+    
+    /**
+     * Whether the given entity is a VTOL
+     * @param e the entity to examine
+     * @return True or false
+     */
+    public static boolean isVTOL(Entity e)
+    {
+    	return e.getEntityType() == Entity.ETYPE_VTOL;
+    }
+    
+    /**
+     * Whether the given entity is a Spheroid dropship
+     * @param e the entity to examine
+     * @return True or false
+     */
+    public static boolean isSpheroidDropship(Entity e)
+    {
+    	return e.isAero() && ((IAero) e).isSpheroid();
+    }
```

Comment:
All `IAero` objects return true for `isAero()`.

Reply:
Hmm, I think I fixed a crash bug relating to this logic recently.

Reply:
Ok, looks like the similar logic is in PathEnumerator ~line 200 (where I check if the unit is an airborne aerodyne unit on a ground map). 

IAeros may return true for isAero, but this method makes no assumptions about the nature of the passed-in entity. I want it to also return false for things that aren't aircraft at all.

Reply:
`IAero` is a common interface for the `Aero` class and `LandAirMech`. `isAero()` returns true for `Aero` and its descendants but only returns true for LAMs when they are in fighter mode.